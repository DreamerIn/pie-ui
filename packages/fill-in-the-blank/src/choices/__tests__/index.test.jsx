import { shallow } from 'enzyme';
import React from 'react';

import { Choices } from '../index';

describe('Choices', () => {
  let w;
  let onChange = jest.fn();
  const wrapper = extras => {
    const defaults = {
      classes: {
        choices: 'choices',
        choicesContainer: 'choicesContainer',
        choiceHolder: 'choiceHolder'
      },
      className: 'className',
      onChange,
      choices: [{ id: '1', content: 'foo' }]
    };
    const props = { ...defaults, ...extras };
    return shallow(<Choices {...props} />);
  };
  describe('snapshot', () => {
    it('renders', () => {
      w = wrapper();
      expect(w).toMatchSnapshot();
    });
  });
});
