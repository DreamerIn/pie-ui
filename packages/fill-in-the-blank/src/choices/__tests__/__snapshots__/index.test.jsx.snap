// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Choices snapshot renders 1`] = `
<div
  className="choices className"
>
  <div
    className="choicesContainer"
  >
    <div
      className="choiceHolder"
      key="0"
    >
      <Wrapped
        content="foo"
        id="1"
      />
    </div>
  </div>
</div>
`;
